packages/mask/src/plugins/RealityCards/SNSAdaptor/MarketView.tsx

```diff
@@ -0,0 +1,356 @@
+            />
+
+            <CardContent className={classes.content}>
+                <Grid container direction="column">
+                    {market.cards.map((card: RCCard) => (
+                        <Grid item key={card.id}>
+                            <CardView card={card} market={market} />
+                        </Grid>
+                    ))}
+                </Grid>
+            </CardContent>
+        </Card>
+    )
+}
+
+function MarketDetails(props: MarketDetailsProps) {
+    const { market } = props
+    const { t } = useI18N()
+
+    const [descriptionDialogOpen, setDescriptionDialogOpen] = useState(false)
+    const [giveawayDialogOpen, setGiveawayDialogOpen] = useState(false)
+    const chainId = useChainId()
+    const token = useBaseToken()
+
+    const [day, setDay] = useState(0)
+    const [hour, setHour] = useState(0)
+    const [minute, setMinute] = useState(0)
+    const state: {
+        label: string
+        color: 'default' | 'error' | 'success' | 'warning' | 'primary' | 'secondary' | 'info' | undefined
+        potSize: string
+    } = useMemo(() => {
+        return market.state === MarketState.Open
+            ? {
+                  label: t('plugin_realitycards_event_state_open'),
+                  color: 'success',
+                  potSize: t('plugin_realitycards_event_pot_size'),
+              }
+            : market.state === MarketState.Withdraw
+            ? {
+                  label: t('plugin_realitycards_event_state_ended'),
+                  color: 'default',
+                  potSize: t('plugin_realitycards_event_final_pot_size'),
+              }
+            : {
+                  label: t('plugin_realitycards_event_state_locked'),
+                  color: 'warning',
+                  potSize: t('plugin_realitycards_event_final_pot_size'),
+              }
+    }, [market.state])
+
+    const avgRental = useMemo(() => {
+        const utcTimestamp = new Date(Date.now() + new Date().getTimezoneOffset() * 60 * 1000).getTime() / 1000
+        const elapsedTime = utcTimestamp - Number.parseInt(market.openingTime, 10)
+        return formatBalance(
+            new BigNumber(market.totalCollected)
+                .minus(market.sponsorAmount)
+                .div(elapsedTime)
+                .multipliedBy(60 * 60),
+            token.decimals,
+            1,
+        )
+    }, [market.openingTime, market.totalCollected, market.sponsorAmount, token.decimals])
+
+    const amountFromRent = useMemo(() => {
+        return new BigNumber(market.totalCollected).minus(market.sponsorAmount)
+    }, [market.totalCollected, market.sponsorAmount])
+
+    useInterval(() => {
+        const date1 = Date.now()
+        const date2 = Number.parseInt(market.lockingTime, 10) * 1000
+        const diff = date2 - date1
+        setDay(new Date(diff).getUTCDate() - 1)
+        setHour(new Date(diff).getUTCHours())
+        setMinute(new Date(diff).getUTCMinutes())
+    }, 10)
```

Comment:
Since the minimize period is `minute`, updating every 10ms is too frequent (even more frequent than UI fps).
I think make it 1000ms (or 1 second) is enough.
```suggestion
    },  1000)
```